app/src/main/java/org/simple/clinic/encounter/EncounterAndObservations.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.encounter
+
+import org.simple.clinic.bp.BloodPressureMeasurement
+
+data class EncounterAndObservations (
```

Comment:
Suggestion, should we just call this `Observations`?

Reply:
We cannot call it that since it would be incorrect.
`Encounter` and `Observations` have a 1:many relationship. This class represents a `Encounter` and all of its `Observations`. I can call it `EncounterWithObservations` or `ObservationsForEncounter`. Wdyt?

Reply:
`ObservationsForEncounter` seems more accurate.